cypress/component/Drawer.spec.tsx

```
@@ -14,6 +73,7 @@ const TestDrawer = () => {
       <Modal
         open={isOpen}
         onClose={hideModal}
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Hm, is this the best way we can deal with the problem?

---

I think we can solve this by narrowing `document.getElementById('modal-container')`, wdyt?

We can just put a `throw Error()` on the start of the function, if it ever get a null value it will at least fail fast and we can easily fix the test:

``` typescript
import assert from 'assert'

const container = document.getElementById('modal-container')

// Typescript actually understands this assert will throw when container is null
// Thanks to https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions
assert(container != null)

// So now you can call it as if it were never null
container.innerHTML = 'foo'
```

---

Ok, I didn't care this much since it is a test file but maybe I should. I applied the way @augustobmoura suggested. Thanks 🙇‍♂️